mmocr/models/textrecog/recognizer/seg_recognizer.py

```diff
@@ -118,14 +120,15 @@ def simple_test(self, img, img_metas, **kwargs):
 
         out_head = self.head(out_neck)
 
-        texts, scores = self.label_convertor.tensor2str(out_head, img_metas)
-
+        # texts, scores = self.label_convertor.tensor2str(out_head)
```

Comment:
It is not used. If it is used, it will cause the constant tensor to enter. When it is called on the C + + side, all the output will become the result of your conversion